Makefile

```diff
@@ -155,5 +155,8 @@ run:
 runlocal:
```

Comment:
Do we need both `runlocal` and `rundev` targets?

Reply:
I have no idea about `runlocal` and where and when it is used, so I didn't dare to remove it.

Reply:
Looks like you [added it 3 years ago](https://github.com/quickfeed/quickfeed/commit/588e9111f7f55a5216c4866cdaac9066c5d8f1be) 😜

Reply:
The one I added was `@aguis -service.url 127.0.0.1 -http.addr :9091 -http.public ./public` and was used to run on the `<username>.itest.run` with port forwarding. 

The one I didn't dare to remove is `@quickfeed -service.url 127.0.0.1` and it couldn't be used without a registered hostname (which I never had). So I still have no idea about its purpose.